presentation-api/controlling-ua/startNewPresentation_success-manual.html

```diff
@@ -39,11 +72,11 @@
 
                         // Check the instance of the connection
                         assert_true(connection instanceof PresentationConnection, 'The connection is an instance of PresentationConnection.');
-                    });
+                    }), function(e) { reject(e); });
+            });
         }, "The presentation was started successfully.");
```

Comment:
Not your fault, but that test label is not very explicit. I would drop it from there and update the `<title>` to something like: "Checking the chain of events when starting a new presentation"